pkgs/top-level/all-packages.nix

```diff
@@ -32651,10 +32651,6 @@ with pkgs;
 
   waybar = callPackage ../applications/misc/waybar { };
 
-  waybar-hyprland = callPackage ../applications/misc/waybar {
```

Comment:
A good practice in such scenarios is to include a line in `aliases.nix` with a throw message, so that people using this deceased expression know how to update their configs.